app/views/win_prize/win_merch.html.erb

```diff
@@ -8,17 +8,19 @@
   </head>
 
   <body>
+    <img src="<%= @server_url %>logoWheel.png" alt="Logo Wheel">
```

Comment:
Just dropping in for a detail: I'd namespace assets with a dash `logo-wheel.png`

Reply:
Ok, I'll fix it

Reply:
Done